### PR TITLE
Switch to next-i18next

### PR DESCRIPTION
**Pull request by sebastien-comeau**
### Description

List of proposed changes:

- Replace `next-translate` with `next-i18next`

### What to test for/How to test

Make sure the localizations load correctly as before.

### Additional Notes

next-translate requires us to add the `/pages` files in our docker image and they are not supose to be there. It's a known issue that I faced in other project with next-translate. Someone oppened an issue in next-translate repo:
https://github.com/aralroca/next-translate/issues/892

next-i18next repo and guide
https://github.com/i18next/next-i18next

### CODE REVIEW

File: pages/status.tsx [hunk 1]
```diff
@@ -1,19 +1,30 @@
...
-import { CheckStatusRequestBody } from './api/check-status'
-import { useCheckStatus } from '../hooks/api/useCheckStatus'
-import { useFormik } from 'formik'
-import * as Yup from 'yup'
-import useTranslation from 'next-translate/useTranslation'
 import StatusInfo from '../components/StatusInfo'
-import { GetStaticProps } from 'next'
-import Router from 'next/router'
+
+const PageWrapper: FC<{ children: JSX.Element }> = ({ children }) => {
```
Comment (s-laugh) [inline pages/status.tsx:19]: This shouldn't be separated out into another component.

File: pages/status.tsx [hunk 2]
```diff
@@ -69,14 +80,6 @@ const Status: FC = () => {
       )
   }, [formik, t])
 
-  // const handleReset: MouseEventHandler<HTMLButtonElement> = useCallback(
-  //   (e) => {
-  //     e.preventDefault()
-  //     formik.resetForm()
-  //   },
-  //   [formik]
-  // )
```
Comment (s-laugh) [inline pages/status.tsx:78]: Please keep this in for now; we might need it with the business requirements.

**Review by s-laugh — CHANGES_REQUESTED** [review]
All the translation changes look good.
If you want to suggest the other changes to status page, please at least put them in a separate PR.

File: pages/status.tsx (hunk 1 above)
Comment (sebastien-comeau) [reply pages/status.tsx:19]: Why not? Othwise I would need to duplicate the following code at 3 occasions (each conditional return)

```tsx
<Layout>
    <h1 className="mb-4">{t('header')}</h1>
    ( // conditional content here }
</Layout>
```

Comment (sebastien-comeau) [reply pages/status.tsx:19]: I will revert it as it was before.

File: pages/status.tsx [hunk 3]
```diff
@@ -87,12 +90,11 @@ const Status: FC = () => {
   )
 
   //if the api failed, fail hard to show error page
-  if (checkStatusError) throw new Error(checkStatusError.message)
+  if (checkStatusError) throw checkStatusError
```
[the remark this answers is not in the document]
Comment (sebastien-comeau) [reply pages/status.tsx:91]: `ApiError` is a class that extends `Error`. Throwing a new Error or the ApiError directly will act the same.

[the remark this answers is not in the document]
Comment (sebastien-comeau) [reply pages/status.tsx]: I've reverted my changes.

**Review by s-laugh — APPROVED** [review]
Looks good, works well!
Thank you for keeping this PR to just the translation. If you want to suggest the other changes made earlier in the PR, please submit another PR and we can chat about it.

**Pull request merged**